src/components/UserSelector.tsx

```diff
@@ -53,36 +54,45 @@ export const UserSelector: React.FC<Props> = ({
           className="button"
           aria-haspopup="true"
           aria-controls="dropdown-menu"
-          onClick={() => {
+          onClick={(event) => {
             setExpanded(current => !current);
+
+            event.stopPropagation();
           }}
         >
           <span>
-            {selectedUser?.name || 'Choose a user'}
+            {user ? user.name : 'Choose a user'}
           </span>
 
           <span className="icon is-small">
-            <i className="fas fa-angle-down" aria-hidden="true" />
+            <i className={`fas fa-angle-${expanded ? 'up' : 'down'}`} aria-hidden="true" />
```

Comment:
it's better to use the classname library for making a dynamic className
please check other places